### ISSUE TITLE
Consider moving to tuxguitar fork

### ISSUE THREAD

**Issue by cbix** [osam-cologne/archlinux-proaudio#329]
https://sourceforge.net/p/tuxguitar/discussion/522984/thread/0a8c900ddd/

**Comment by jujudusud** [comment]
Moving to the fork and use the latest 1.6.0 version.
The new website is here: [https://www.tuxguitar.app/](https://www.tuxguitar.app/)

**Comment by DanielGibson** [comment]
Yes, this would be great, that fork is now at version 1.6.4 and has lots of fixes and improvements, as far as I can tell.

**Comment by SpotlightKid** [comment]
Should we take a new look at this?
